src/codegen/codegen_ispc_visitor.cpp

```diff
@@ -63,6 +63,39 @@ void CodegenIspcVisitor::visit_var_name(ast::VarName& node) {
     CodegenCVisitor::visit_var_name(node);
 }
 
+void CodegenIspcVisitor::visit_local_list_statement(ast::LocalListStatement& node) {
+    if (!codegen) {
+        return;
+    }
+    auto type = CodegenCVisitor::local_var_type() + " ";
+    printer->add_text(type);
+    auto local_variables = node.get_variables();
+    std::set<std::string> uniform_variables;
+    auto dt_saved_value_exists = false;
+
+    /// Remove dt_saved_value from local_variables if it exists
+    for (auto it_local_variables = local_variables.begin();
+         it_local_variables < local_variables.end();
+         ++it_local_variables) {
+        if ((*it_local_variables)->get_node_name() == "dt_saved_value") {
```

Comment:
I would use a constant instead of the hardcoded string `dt_saved_value`.

Reply:
Good point. I added it as a constant and some documentation on how this name is generated 👍